misc/json-to-record-converter/src/main/java/io/ballerina/converters/util/Constants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.converters.util;
+
+/**
+ * Contains keywords and other constants.
+ *
+ * @since 2201.2.0
+ */
+public class Constants {
+
+    private Constants() {}
```

Comment:
We can remove this constructor since It is not required to explicitly define a constructor.

Reply:
It's a private constructor, and are we still not required to define it? @lnash94